ci-operator/templates/openshift/openshift-azure/build-node-image.yaml

```diff
@@ -125,13 +125,13 @@ objects:
         trap 'rc=$?; if [[ $rc -ne 0 ]]; then
           touch /tmp/shared/exit;
         fi;
-        az group delete -n ${RESOURCE_GROUP} -y --no-wait
+        az group delete -n ${RESOURCEGROUP} -y --no-wait
```

Comment:
we cant use `make delete` here because its ansible image :/

Reply:
`make delete` doesn't make sense here since this is the rg used for building the image.

Reply:
:) just left comment if somebody would want to pick on this :D

Reply:
actually can you rename this to IMAGEBUILD_RESOURCEGROUP? Otherwise it is confusing that we are using the same env as during cluster deployment.

Reply:
We use the same variable later on for tests by adding `${RESOURCEGROUP}-e2e`. this will cause changes in `/tmp/shared/environment` and more confusion. 

Maybe better  `${RESOURCEGROUP}-node-build` and  `${RESOURCEGROUP}-e2e` to keep the format and distinguish type?

Reply:
example last commit

Reply:
OK, at least set `RESOURCEGROUP` to `RESOURCEGROUP_PREFIX`, otherwise it's still confusing the way you have it now.

Reply:
I think done. How about now?